src/amo/components/FeaturedCollectionCard/index.js

```diff
@@ -22,20 +27,30 @@ export default class FeaturedCollectionCard extends React.Component<Props> {
       className,
       footerText,
       header,
+      isTheme,
       loading,
       slug,
       username,
     } = this.props;
 
+    const themeProps = {};
```

Comment:
Same comment as above. This is making it look like passing an `isTheme` to `LandingAddonsCard` would be the best option, as then you could remove logic from this and from `Category`.

Reply:
I think I like the placeholder way that you suggested above.